components/merkledb/tests/common/mod.rs

```diff
@@ -19,18 +19,54 @@ use proptest::test_runner::TestCaseResult;
 
 use std::rc::Rc;
 
-use exonum_merkledb::{Database, Fork, TemporaryDB};
+use exonum_crypto::{hash, Hash, HASH_SIZE};
+use exonum_merkledb::{BinaryKey, Database, Fork, ObjectHash, TemporaryDB};
 
 // Max size of the generated sequence of actions.
+#[allow(dead_code)]
 pub const ACTIONS_MAX_LEN: usize = 100;
 
+#[derive(Debug, Copy, Clone, Eq, PartialEq, Ord, PartialOrd, Hash)]
+pub struct Key(pub [u8; HASH_SIZE]);
+
+impl ObjectHash for Key {
+    fn object_hash(&self) -> Hash {
+        hash(&self.0)
+    }
+}
+
+impl From<[u8; HASH_SIZE]> for Key {
+    fn from(key: [u8; HASH_SIZE]) -> Self {
+        Self(key)
+    }
+}
+
+impl BinaryKey for Key {
+    fn size(&self) -> usize {
+        HASH_SIZE
+    }
+
+    fn write(&self, buffer: &mut [u8]) -> usize {
+        buffer.copy_from_slice(&self.0);
+        self.0.len()
+    }
+
+    fn read(buffer: &[u8]) -> Self::Owned {
+        let mut buf = [0; 32];
+        buf.copy_from_slice(&buffer);
+        Self(buf)
+    }
+}
+
 pub trait FromFork {
     fn from_fork(fork: Rc<Fork>) -> Self;
     fn clear(&mut self);
 }
 
+#[allow(dead_code)]
```

Comment:
Why?

Reply:
As far as I understand every module in external tests directory links with merkledb lib separately.  Not all code in `common.rs` used in all tests and that's why i need to mark it with `dead_code`.

Reply:
Maybe then add a comment, so developer who'll see it in the future won't attempt to remove it?